fix(links): fix links that were valid in the previous doc

educhastenier opened the pull request:
before the new Doc infra & tooling refactoring

educhastenier committed:
fix(links): fix links that were valid in the previous doc

before the new Doc infra & tooling refactoring

alachambre commented on modules/ROOT/pages/living-application-layout.adoc:
```diff
@@ -23,6 +23,8 @@ You can customize this layout by following the xref:customize-layouts.adoc[custo
 
 As of version 7.5.x, the default provided layout does not allow this kind of customization anymore (at least not so easily), but if you start from a 7.4.x Layout you can follow the 7.4 documentation link:/7.4?page=customize-living-application-layout#toc3[customize-living-application-layout] to build a side menu layout.
 
+*FIXME*: The above paragraph is not relevant anymore, as the linked page is no more available, please update it.
```
@julienmege, king of the layouts, could you please check this part?

julienmege replied:
As in the corresponding documentation (in 7.4 archive) the html exemple looks to be corrupted 
![image](https://user-images.githubusercontent.com/4995528/126457327-1b88b466-dad9-4888-938d-8de56f882fb0.png)
, I propose to remove this paragraph, until someone document a new way to create a side Menu using the latest version of the Layout. (We have already done this with Presales team, but not documented)

educhastenier replied:
Perfect thx

julienmege committed:
fix(links): Remove side layout example

As it contains broken link to 7.4 archive documentation,

we propose to remove this paragraph, until we document a new way to create a side Menu using the latest version of the Layout.
